ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -88,11 +88,11 @@ - (instancetype)init {
   resolve([NSNull null]);
 }
 
-RCT_EXPORT_METHOD(signInSilently:(RCTPromiseResolveBlock)resolve
+RCT_EXPORT_METHOD(restorePreviousSignIn:(RCTPromiseResolveBlock)resolve
```

Comment:
are you sure this still works?

Reply:
After some tests, it's worked correctly (after 91c3fdd, f914185)
Tested on this app: https://github.com/D1verGW/react-native-google-signin-test-app
With environment: iOS 12, android api v28

Reply:
Please keep the previous name - signInSilently. Thanks